projects/core/src/lazy-loading/facade-factory/facade-factory.service.ts

```diff
@@ -119,6 +121,17 @@ export class FacadeFactoryService {
       result[property] = this.get(resolver$, property as string);
     });
 
+    result[PROXY_FACADE_INSTANCE_PROP] = true;
+
     return result;
```

Comment:
line 115:  `const result: any = new (class extends (facade as any) {})();` that means proxy is `extends` facade, while our facade service implements the facade (e.g. `export class MultiCartService implements MultiCartFacade`). So, we can use `instanceof` to check whether it is proxy or real facade service.
```
this.injector
      .get(ActiveCartFacade)
      .subscribe((value) =>
        console.log('facade: ', value, value instanceof ActiveCartFacade)
      );
```